tools/regtesting/do_regtest

```diff
@@ -21,7 +21,7 @@ skip_dirs[1]=""
 ndirstorestrict=0
 restrict_dirs[1]=""
 default_err_tolerance="1.0E-14"
-mpiexec="mpiexec --bind-to none"
+mpiexec="mpiexec --bind-to-none"
```

Comment:
Care to elaborate? `--bind-to none` is what is documented by OpenMPI

Reply:
Sry, I forgot about it.